tests/integration/test_web_client.py

```diff
@@ -183,6 +186,31 @@ def test_creation_lifecycle(self, antares_web: AntaresWebDesktop):
         assert storage_fr.name == st_storage_name
         assert storage_fr.id == "cluster_test"
 
+        # testing
+        thermal_service = ThermalApiService(api_config, study_id=study.service.study_id)
+        renewable_service = RenewableApiService(api_config, study_id=study.service.study_id)
+        storage_service = ShortTermStorageApiService(api_config, study_id=study.service.study_id)
```

Comment:
Here you shouldn't introduce these services. It's an implementation detail that the user shouldn't know about. What the user will do is **area_fr.read_thermal_clusters()** which will call the thermal service but he doesn't have to know about this. This goes for the 3 services you introduced.